vite.config.ts

```diff
@@ -20,11 +26,16 @@ export default defineConfig(({ command, mode }) => {
           target: `${env.BACKEND_URL}`,
           changeOrigin: true,
         },
+        "/v2": {
+          target: `${env.BACKEND_URL}`,
+          changeOrigin: true,
+        },
         "/graphql": {
           target: `${env.BACKEND_URL}`,
           changeOrigin: true,
         },
       },
+      https: true,
```

Comment:
Please update the README to indicate that we're using https in dev mode (perhaps with the why), and preparing the developer for the fact that, since the cert is self-signed, they will need to proceed through the certificate warning.